lib/srv/usermgmt.go

```diff
@@ -515,7 +567,12 @@ func (u *HostUserManagement) createGroupIfNotExist(group string) error {
 		return nil
 	}
 
-	return trace.Wrap(err, "%q", group)
+	if err != nil {
+		return trace.Wrap(err, "%q", group)
```

Comment:
I'm almost certain we reduced the context there because the resulting error message was really repetitive. Something like `creating configured groups: group: "my-group": not a valid group name`, whereas dropping the context here gives you `creating configured groups: "my-group" not a valid group name` which is a little easier to visually parse.